tests/test_infra_oncall.py

```diff
@@ -1,13 +1,59 @@
+import datetime
+
 import httpx
+import pytest
 
 
-async def test_oncall_empty(bot, plugin):
+async def test_oncall_alias(bot, plugin):
     await bot.send("!oncall")
     assert len(bot.sent) == 1
     expected = "No one from Fedora Infrastructure is currently on call"
     assert bot.sent[0].content.body == expected
 
 
+async def test_oncall_alias_with_arg(bot, plugin):
+    await bot.send("!oncall add ryanlerch")
+    assert len(bot.sent) == 1
+    expected = (
+        "`!oncall` is an alias to `!infra oncall list` please use "
+        "the `!infra oncall` command for changing the oncall list"
+    )
+    assert bot.sent[0].content.body == expected
+
+
+async def test_oncall_empty(bot, plugin):
+    await bot.send("!infra oncall list")
+    assert len(bot.sent) == 1
+    expected = "No one from Fedora Infrastructure is currently on call"
+    assert bot.sent[0].content.body == expected
+
+
+@pytest.mark.parametrize("listcommands", ["!oncall", "!infra oncall list"])
+async def test_oncall_list(bot, plugin, db, monkeypatch, listcommands):
+    SET_TIME = datetime.datetime(2023, 11, 1, 13, 00, 00)
+
+    class thedatetime:
+        @classmethod
+        def now(cls, t):
+            return SET_TIME
+
+    monkeypatch.setattr(datetime, "datetime", thedatetime)
```

Comment:
You've created a `freeze_datetime` fixture, should it be used here?

Reply:
whoops! i left that lying around -- i couldnt get it working as a fixture, so i just did it here, let me go back and see if i can get the fixture working!

Reply:
okay, its using the fixture now! sorry about that!